includes/admin/product-settings.php

```diff
@@ -187,7 +187,7 @@ public static function save_subscription_fields($post_id)
         $possible_periods = ['d', 'w', 'm', 'y'];
 
         // Get and validate data
-        $test_mode           = (!empty($_POST['mbbxs_test_mode']) ? $_POST['mbbxs_test_mode'] : '');
+        $test_mode           = (!empty($_POST['mbbxs_test_mode']) == 'yes');
```

Comment:
Esta condición siempre da false
```suggestion
        $test_mode           = isset($_POST['mbbxs_test_mode']) && $_POST['mbbxs_test_mode'] === 'yes';
```